packages/client/Atmosphere.ts

```diff
@@ -110,8 +111,13 @@ export default class Atmosphere extends Environment {
       body
     })
     const contentTypeHeader = res.headers.get('content-type') || ''
-    const contentType = contentTypeHeader.toLowerCase()
-    return contentType.startsWith('application/json') ? res.json() : null
+    if (contentTypeHeader.toLowerCase().startsWith('application/json')) {
+      return res.json()
+    }
+    if (res.status === 401) {
```

Comment:
this will be hit when someone tries to hit a route with an expired token (ie the fetch request was sent before the check for the websocket)